### PR TITLE
New Card

### PR DESCRIPTION
**Pull request by dquindara8**
Add my card information

### CODE REVIEW

**Review by Preet-Sojitra — APPROVED** [review]
LGTM. 
Your PR will be merged soon.

File: index.html [hunk 1]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:118]:
```suggestion
        <!-- ________ *Quindara* Contributor card START ________  -->
```

File: index.html [hunk 2]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="https://developer.mozilla.org/en-US/docs/Learn" target="_blank" title="Learn Web Development">MDN</a>
+              </li>
+              <li>
+                <a href="https://www.learn-html.org/" target="_blank" title="Learn HTML">Learn-html.org</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:141]:
```suggestion
        <!-- ________*Quindara* Contributor card END ________  -->
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@dquindara8, Thanks for your contribution. Kindly do the required changes and commit, it will be merged soon!

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@dquindara8, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**